src/stories/Forms/LoginForm/LoginForm.stories.js

```diff
@@ -107,20 +111,20 @@ export const ExampleLoginWithBug = {
     // await expect(message).toHaveStyle("color: rgb(255, 0, 0);");
   },
 };
-export const ExampleApi = () => {
-  setupApiMock();
-  const expectedResponse = [
-    { id: 1, name: 'Alice' },
-    { id: 2, name: 'Bob' },
-    { id: 3, name: 'Charlie' },
-  ];
-
-  // Call example API endpoint
-  axios.get('/api/users').then((response) => {
-    console.log(response.data);
-    expect(response.status).toBe(200);
-    expect(response.data).toEqual(expectedResponse);
-  });
-
-  return <div>Check the console for API response</div>;
-};
\ No newline at end of file
+// export const ExampleApi = () => {
+//   setupApiMock();
+//   const expectedResponse = [
+//     { id: 1, name: 'Alice' },
+//     { id: 2, name: 'Bob' },
+//     { id: 3, name: 'Charlie' },
+//   ];
+//
+//   // Call example API endpoint
+//   axios.get('/api/users').then((response) => {
+//     console.log(response.data);
+//     expect(response.status).toBe(200);
+//     expect(response.data).toEqual(expectedResponse);
```

Comment:
Please add similar verifications and then remove commented lines

Reply:
No need for it, answered in slack chanel, not releted to the actual changes

Reply:
Then lets leave this story uncommented, it will not be redundant I think